index.js

```diff
@@ -0,0 +1,81 @@
+import { i18n } from '@kbn/i18n';
```

Comment:
For future ref. for all files- please add the copyright notice at the beginning of the file. Refer to https://github.com/opendistro-for-elasticsearch/kibana-reports/blob/dev/public/components/main/index.ts